samples/csv/input/input_queries.csv

```diff
@@ -91,3 +91,26 @@ where
      ref like '%master%'
      and repo_name = 'cdnjs/cdnjs';
 "
+
+9, "SELECT
```

Comment:
Can we please use a simpler example here. (See comment in readme)

Reply:
Done